beesdoo_shift/models/cooperative_status.py

```diff
@@ -1,11 +1,13 @@
 import logging
-from datetime import timedelta
+from datetime import datetime, timedelta
 
 from odoo import _, api, fields, models
 from odoo.exceptions import UserError, ValidationError
 
 _logger = logging.getLogger(__name__)
 
+# todo rename mentions to cooperatives
+
```

Comment:
If this still needs to be done in this PR: There's a TODO here that still needs doing :)

If this is more long-term: I think it's good practice to create an issue for this.

Same for other TODOs.

Reply:
It's a long term todo. We left a lot of long-term todo in the code but I agree with your suggestion. But since the code will be moved to `shift-planning` repository, we would need to migrate all issues as well. What do you think we should do ?
- create issue here then migrate all issues ?
- create issues in `shift-planning` directly although code does not exist ?
- leave the comments and create issues as we move the modules to `shift-planning` ?

Reply:
I think creating the issues in shift-planning directly is probably fine, even though the code is not there yet. I would probably create the issues, then edit the TODO comments to refer to the issues (full URL).

Reply:
I listed the todo in the general project task : https://github.com/coopiteasy/vertical-cooperative/issues/204#issue-1121003036

Reply:
and removed the todo